Library/QuickPulseStateManager.ts

```diff
@@ -124,14 +125,15 @@ class QuickPulseStateManager {
         }
 
         let currentTimeout = QuickPulseStateManager._isCollectingData ? 1000 : 5000;
-        if (QuickPulseStateManager._isCollectingData && Date.now() - this._lastSuccessTime >= 20000) {
+        if (QuickPulseStateManager._isCollectingData && Date.now() - this._lastSuccessTime >= 20000 && !this._lastSendSucceeded) {
```

Comment:
What's the logic here for what's a static vs instance level property?

Reply:
Moved to have everything as instance level